src/common/options/global.yaml.in

```diff
@@ -1880,6 +1880,15 @@ options:
   services:
   - mon
   with_legacy: true
+- name: mon_config_history_size
+  type: size
```

Comment:
Should this maybe be `uinit` or `int`? What happens if `0` or `-1` are entered?  Since various other options have special behaviors for those values, users might enter them out of habit.  Do we *want* an option to configure the current, unlimited behavior?    I'm half tempted to suggest a separate setting for osd_memory_target given the autotuner, but I really hate hardcoding like that.